modules/Bio/EnsEMBL/Analysis/Hive/Config/SanityChecksStatic.pm

```diff
@@ -140,6 +140,118 @@ sub _master_config {
    },
 
   'gene_db_checks' => {
+    'bird_basic' => {
+      'genblast' => {
+        'logic_names' => {
+          'genblast'            => 200000,
+          'genblast_not_best'   => 250000,
+        }, # logic_names
+        'biotypes' =>    {
+          'human_pe12_'         => 20000,
+          'aves_pe12_'          => 15000,
+          'mammals_pe12_'       => 100000,
+          'reptiles_pe12_'      => 1000,
+        }, # biotypes
+      }, # genblast
+      'projection_coding' => {
+        'logic_names' => {
+          'project_transcripts' => 30000,
+        }, # logic_names
+        'biotypes' =>    {
+          'projection'          => 10000,
+        }, # biotypes
+      }, # projection_coding
+      'rnaseq_blast' =>  {
+        # This one is an issue, logic names, counts are varied and one biotype is a
+        # substring of the other. At the moment it's really just a check that theres'
+        # some stuff in there
+        'logic_names' => {
+#          $species.'_merged_rnaseq' => 10000,
+        }, # logic_names
+        'biotypes' =>    {
+          'rnaseq'              => 10000,
+         }, # biotypes
+      }, # rnaseq_blast
+      'layer' => {
+        'logic_names' =>    {
+        }, # logic_names
+        'biotypes' =>    {
+          'human_pe12_'          => 0,#not sure what value to set
```

Comment:
Was this really variable in birds, or was this just copied from mammals?

Reply:
This was followed on from mammals. I excluded some the self biotypes following your comment about this not always being available.